third_party/blake3/blake3.BUILD

```diff
@@ -57,6 +57,9 @@ cc_library(
             # lacking the headers to compile AVX512.
 	    "-DBLAKE3_NO_AVX512",
 	],
+	"@bazel_tools//src/conditions:linux_aarch64": [
+            "-DBLAKE3_USE_NEON=0",
+        ],
```

Comment:
Shouldn't we instead include `c/blake3_neon.c` like we do in the `@bazel_tools//src/conditions:darwin_arm64` condition above, and set `-DBLAKE3_USE_NEON=1`? Same for windows arm64 @meteorcloudy.

Reply:
Yeah that works, I can make the change for windows here as well, but I'll wait for @meteorcloudy comments.

Reply:
LGTM, /cc @tylerwilliams who made the blake3 contribution in https://github.com/bazelbuild/bazel/pull/18682

Reply:
Tyler gave me the tip. I think we should do the Windows path as well.

Reply:
Yeah this change looks good to me too